.github/workflows/main.yaml

```
@@ -79,6 +79,11 @@ jobs:
   cli-tests:
     needs: cli-test-shards
     if: ${{ always() }}
+    permissions:
+      actions: read
+      code-quality: write
```

actionlint (1.7.12)

[error] unknown permission scope "code-quality". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)